Do not launch workflows for PRs w/o "can be tested"

Felixoid opened the pull request:
<!---
A technical comment, you are free to remove or leave it as it is when PR is created
The following categories are used in the next scripts, update them accordingly
utils/changelog/changelog.py
tests/ci/cancel_and_rerun_workflow_lambda/app.py
-->
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)


### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Our CI used to approve runs w/o `can be tested` label. It's fixed here

Felixoid committed:
Do not launch PRs w/o "can be tested"

robot-ch-test-poll3 commented:
<!-- automatic status comment for PR #49726 from ClickHouse/ClickHouse:require-can-be-tested -->
This is an automated comment for commit f6372dd3fc1ceb1256b7aec37e86559e405c7cf9 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/49726/f6372dd3fc1ceb1256b7aec37e86559e405c7cf9/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

Felixoid committed:
Replace old release with new packages in suspicious patterns

Felixoid commented:
The lambda is already deployed

Felixoid committed:
Print debugging information in workflow_jobs_lambda

Felixoid committed:
Upgrade all lambdas runtime to 3.10, fix requirements

The recent update of urllib3 completely broken the world

- https://github.com/urllib3/urllib3/issues/2168
- https://github.com/boto/botocore/issues/2926
- https://github.com/psf/requests/issues/6443

Felixoid commented:
I've found that the recent lambda deployment has wholly broken it. The details are in the following tickets:

- urllib3/urllib3#2168
- boto/botocore#2926
- psf/requests#6443
